marimo/_runtime/packages/pypi_package_manager.py

```diff
@@ -86,5 +111,16 @@ async def install(self, package: str) -> bool:
 class RyePackageManager(PypiPackageManager):
     name = "rye"
 
+    def is_manager_installed(self) -> bool:
+        try:
+            return (
+                subprocess.run(
+                    ["rye", "--help"], capture_output=True
+                ).returncode
+                == 0
+            )
+        except Exception:
+            return False
```

Comment:
seems like `[self.name, "--help"]` is pretty standard - maybe we just included that in the super class?

Reply:
Good idea. Also made the implementation more robust.